articles/app-service/app-service-plan-manage.md

```diff
@@ -42,7 +42,7 @@ You can create an empty App Service plan, or you can create a plan as part of ap
 You can move an app to another App Service plan, as long as the source plan and the target plan are in the _same resource group and geographical region_.
 
 > [!NOTE]
-> Azure deploys each new App Service plan into a deployment unit, internally called a webspace. Each region can have many webspaces, but your app can only move between plans that are created in the same webspace. An App Service Environment is an isolated webspace, so apps can be moved between plans in the same App Service Environment, but not between plans in different App Service Environments.
+> Azure deploys each new App Service plan into a deployment unit, internally called a webspace. Each region can have many webspaces, but your app can only move between plans that are created in the same webspace. An App Service Environment can have multiple webspaces, your app can only move between plans that are created in the same webspace.
```

Comment:
"An App Service Environment can have multiple webspaces, **but** your app can only move between plans that are created in the same webspace."